2016.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var maximumDifference = function (nums) {
+  let result = 0;
+
+  for (let i = 0; i < nums.length; i++) {
+    for (let j = i + 1; j < nums.length; j++) {
+      if (nums[j] > nums[i]) {
+        diff = nums[j] - nums[i];
+        if (diff > result) {
+          result = diff;
+        }
+      }
```

Comment:
Too much nesting makes your code look complicated.
Let's try to reduce the number of nesting like this.
```suggestion
     if (nums[j] <= nums[i]) {
       continue;
     }
    const diff = nums[j] - nums[i];

    if (diff > result) {
      result = diff;
    }
```

Reply:
You can also assign the largest number of multiple values to `result` like this.
This way is useful and makes your code look cleaner because it's just one line of code.
```
result = Math.max(result, diff)
```
Ref. [Math.max()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max)